### PR TITLE
feat(discover) Add sum aggregation to snuba APIs

### PR DESCRIPTION
**Pull request by evanh**
Add the sum aggregation to the query builders and APIs.

### CODE REVIEW

File: src/sentry/api/event_search.py [hunk 1]
```diff
@@ -722,6 +722,7 @@ def to_list(value):
     "min": {"snuba_name": "min", "fields": ["time", "timestamp", "transaction.duration"]},
     "max": {"snuba_name": "max", "fields": ["time", "timestamp", "transaction.duration"]},
     "avg": {"snuba_name": "avg", "fields": ["transaction.duration"]},
+    "sum": {"snuba_name": "sum", "fields": ["time", "timestamp", "transaction.duration"]},
```
Comment (markstory) [inline src/sentry/api/event_search.py]: I'm not sure suming up the time or timestamp columns is as useful as the duration. We might want to start with only duration.

Comment (dashed) [reply src/sentry/api/event_search.py]: I agree. timestamp are just unix timestamps; and summing them up isn't actually useful.

Just having `["transaction.duration"]` for the fields would be sufficient.

**Comment by dashed** [comment]
This works great! 🎉 

Screen Shot 2019-12-11 at 1 30 49 PM

-----

@markstory  and I have just one minor feedback; otherwise, this PR is good to go.

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Comment by dashed** [comment]
@evanh thanks for looking into this; especially for the frontend side. 

Updating multiple files just to add this aggregate function probably wasn't ideal 😬. I'm hoping to refactor this a bit later on.

**Pull request merged**